### PR TITLE
New questions/sections UI component

### PR DESCRIPTION
**Pull request by ahmedhamodi**
I just realized that both Xin's task (https://www.notion.so/uwblueprintexecs/Active-Epics-95f65fd1b589465e9adbe5ead3815e2e?p=318c4ac5920a400ead8f4a93dc9fdba9) and my task (https://www.notion.so/uwblueprintexecs/Active-Epics-95f65fd1b589465e9adbe5ead3815e2e?p=34df7cd158c444109da12137b4cf1c9f) both would need this component.

So probably a good idea to get this merged separately, so we can both work on getting the functionality implemented. Also, if Xin hasn't begun work yet, I can take her task off her hands since it's quite similar in logic. Let me know what you think @GodGreg @xinhaoz

### CODE REVIEW

File: src/Components/FormCreation/AddCardComponent.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  const classes = useStyles();
+
+  return (
+    <div>
+      <Card className={classes.root}>
+        <CardContent classes={{ root: classes.content }}>
+          <p className={classes.text}>
+            <span style={{ cursor: "pointer", color: "#2261AD" }}>
+              <FontAwesomeIcon
+                style={{
```
Comment (ahmedhamodi) [inline src/Components/FormCreation/AddCardComponent.js:49]: TODO: Refactor inline css styling

File: src/Components/FormCreation/AddCardComponent.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    marginBottom: 16
+  },
+  content: {
+    marginLeft: 10,
+    padding: 0
+  },
+  expandOpen: {
+    transform: "rotate(180deg)"
+  },
+  root: () => ({
```
Comment (GodGreg) [inline src/Components/FormCreation/AddCardComponent.js:19]: Is there a reason this a function? (not that it is bad, just curious)  

I normally do it with just `{ }`

**Review by GodGreg — APPROVED** [review]
LGTM

File: src/Components/FormCreation/AddCardComponent.js (hunk 2 above)
Comment (ahmedhamodi) [reply src/Components/FormCreation/AddCardComponent.js:19]: I had copied that from a different class. It's not needed here so I'll remove it when I do the css refactoring. I think for that class it had some variables being used in the css so it was needed? Not sure.

**Pull request merged**